src/video-recorder/process.js

```diff
@@ -0,0 +1,104 @@
+import { spawn } from 'child_process';
+import { flatten } from 'lodash';
+import Promise from 'pinkie';
+import AsyncEmitter from '../utils/async-event-emitter';
+import delay from '../utils/delay';
+
+
+const DEFAULT_OPTIONS = {
```

Comment:
It'd be great to add a comment about chosen options, because it looks like a spell now =).